backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.test.ts

```diff
@@ -42,7 +49,11 @@ describe("Async Issue Biometric Credential", () => {
 
   beforeEach(() => {
     dependencies = {
-      env: {},
+      env: {
+        BIOMETRIC_VIEWER_ACCESS_KEY: "mock_biometric_viewer_access_key",
```

Comment:
nitpick: same as above and entirely non blocking

```suggestion
        BIOMETRIC_VIEWER_ACCESS_KEY: "mockBiometricViewerAccessKey",
```

Reply:
Was wondering about this myself, on the ticket it mentions 'mock_biometric_viewer_access_key' as the param value, and it's implemented like that in biometricTokenHandler.test.

Is it something we recently changed targeting this specifically? Happy to discuss

Reply:
> Is it something we recently changed targeting this specifically? Happy to discuss

Not sure actually might be worth asking whoever did that. If that was me, I didn't do it on purpose haha

Reply:
The actual value of the parameter we put into SMPS in dev and build should be 'mock_biometric_viewer_access_key' (as that's what the readid mock will accept I think). It doesn't matter what value we use in the test here, so long as it matches the mock set up above.